common/runtime/src/main/java/io/quarkiverse/googlecloudservices/common/GcpBootstrapConfiguration.java

```diff
@@ -13,9 +13,17 @@
  */
 @ConfigRoot(name = "google.cloud", phase = ConfigPhase.BOOTSTRAP)
 public class GcpBootstrapConfiguration {
+
+    /**
+     * Enabled set projectId by defaultV value, otherwise null.
+     *
+     */
+    @ConfigItem()
+    public boolean defaultProjectIdEnabled = true;
```

Comment:
I wonder if in the future we may want to use ServiceOptions to get "something" else, the Metadata server can retrieve other configuration options (like the region) and we may need it some day.

This to say that we may consider a different name for this option, like enableMetadataServer or loadDefaultFromMetadataServer or loadDefaultFromServiceOptions ...

Reply:
ok, I agree with you.